Formula/percona-xtrabackup.rb

```diff
@@ -8,8 +8,8 @@ class PerconaXtrabackup < Formula
   revision 2
 
   livecheck do
-    url "https://www.percona.com/downloads/Percona-XtraBackup-LATEST/"
-    regex(/value=.*?Percona-XtraBackup[._-]v?(\d+(?:\.\d+)+-\d+)["' >]/i)
+    url "https://docs.percona.com/percona-xtrabackup/8.0/index.html"
+    regex(/href=.*?v?(\d+(?:\.\d+)+(?:-\d+)?)(?:\.\d+)?\.html/i)
```

Comment:
```suggestion
    url "https://docs.percona.com/percona-xtrabackup/latest/index.html"
    regex(/href=.*?v?(\d+(?:[.-]\d+)+)\.html/i)
    strategy :page_match do |page, regex|
      page.scan(regex).map do |match|
        # Convert a version like 1.2.3-4.0 to 1.2.3-4 (but leave a version like
        # 1.2.3-4.5 as-is).
        match[0].sub(/(-\d+)\.0$/, '\1')
      end
    end
```

Same feedback regarding the `url` and `regex` as in https://github.com/Homebrew/homebrew-core/pull/133972/files#r1232873035

Though this is `percona-xtrabackup`, there have been `percona-server` versions like [`5.5.58-38.10`](https://github.com/Homebrew/homebrew-core/pull/21458) in the past where the minor version for the suffix was significant, so I don't think we can simply omit it from the capture group. I think we'll have to use a `strategy` block to modify the ones that end in `.0` and leave the others (e.g., `-38.10`) as-is.